services/metadactyl-clj/src/metadactyl/persistence/oauth.clj

```diff
@@ -117,10 +116,10 @@
   (let [id  (if (string? id) (UUID/fromString id) id)
         req (get-authorization-request id)]
     (when (nil? req)
-      (throw+ {:error_code ce/ERR_BAD_REQUEST
-               :reason     (str "authorization request " (str id) " not found")}))
+      (throw+ {:type  :clojure-commons.exception/not-found
```

Comment:
I agree that it should be a 400 response. I think the original error code ERR_BAD_REQUEST would have cause a 400 response too.